### ISSUE TITLE
Images are missing in "master/modules/perception/camera/README.md"

### ISSUE THREAD

**Issue by Deephome** [ApolloAuto/apollo#14770]
For example, the figure of the network structure is missing at this link:

https://github.com/ApolloAuto/apollo/blob/master/docs/specs/images/3d_obstacle_perception/camera_network.png

**Comment by daohu527** [comment]
Yeah, thank you for finding this problem, we will use tools to check for dead links!
